org.eclipse.scanning.sequencer/src/org/eclipse/scanning/sequencer/AcquisitionDevice.java

```diff
@@ -437,21 +432,22 @@ private void close(boolean errorFound, IPosition last) throws ScanningException
 				nexusScanFileManager.scanFinished(); // writes scanFinished and closes nexus file
 
 				// We should not fire the run performed until the nexus file is closed.
-			// Tests wait for this step and reread the file.
-		    fireRunPerformed(last);              // Say that we did the overall run using the position we stopped at.
-
+				// Tests wait for this step and reread the file.
+				fireRunPerformed(last); // Say that we did the overall run using the position we stopped at.
 			} finally {
-			// only fire end if finished normally
-			if (!errorFound) fireEnd(last);
+				// only fire end if finished normally
+				if (!errorFound)
+					fireEnd(last);
 			}
-
 		} finally {
-	    try {
+			try {
 				annotationManager.invoke(ScanFinally.class, last);
-			} catch (IllegalAccessException | IllegalArgumentException | InvocationTargetException | InstantiationException | EventException e) {
+			} catch (IllegalAccessException | IllegalArgumentException | InvocationTargetException
+					| InstantiationException | EventException e) {
 				throw new ScanningException(e);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this throw statement from this finally block. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1143)